### DISCUSSION TITLE
Adding content to junction field via sdk

### DISCUSSION THREAD

**Discussion by torstenlemke** [directus/directus#10028] [Archive]
Hey Guys :)

I'm trying to create an item which has a field which is a m2m field. 
Sadly I'm not too sure what information I'd need to put into this field for it to create the junction item.

Currently I'm getting the following error : 

```php
Array ( [errors] => Array ( [0] => Array ( [message] => Invalid one-to-many update structure: "value" must be of type object [extensions] => Array ( [code] => INVALID_PAYLOAD ) ) ) ) 
```

( i know this is php array but you will get the point :) )

This is a obviously an error in my syntax but I'm not sure what i need to do 

Thanks in advance :)
Regards Torsten

**Comment by rijkvanzanten** [comment]
> This is a obviously an error in my syntax but I'm not sure what i need to do

What is your syntax now? Little tricky to tell you what's wrong with the used code if it's not shown 😄 

Have you seen https://docs.directus.io/reference/introduction/#one-to-many-many-to-many? That details what values to nest in modifications
